apps/portal/src/modules/collections/actions/helpers.ts

```
@@ -120,14 +121,15 @@ export function validateCollectionData(
- * @param transaction - Database transaction (optional)
- * @returns Promise resolving to upserted settings
+ * Inserts or updates collection settings for a given collection.
+ *
+ * If settings for the specified collection ID exist, updates them; otherwise, inserts new settings. Returns the upserted settings or null if the operation fails.
+ *
+ * @returns The upserted collection settings, or null if the operation did not succeed.
  */
 export async function upsertCollectionSettings(
   settingsData: CollectionSettingsData,
-  transaction: any = db,
+  transaction: any = db
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)